src/main/java/at/pfeifer/chatapp/controller/ChatController.java

```diff
@@ -28,21 +28,20 @@ public class ChatController implements Initializable {
     @FXML
     private Button sendButton;
 
-    @FXML
-    private TextField userName;
-
-    String userNameText;
+@FXML
+private TextField signatureInput;
 
     @FXML
     void sendMessage() {
         String message = messageInput.getText();
 
         if (message.isEmpty()) return;
-        String messageWithUserName = combineUserNameWithMessage(message, userNameText);
+        String messageWithSignature= combineUserNameWithMessage(message, signatureInput.getText());
```

Comment:
The name of the function is wrong should be
```suggestion
String messageWithSignature= combineMessageWithSignature(message, signatureInput.getText());
```

Reply:
that is outdated, I already changed it